### PR TITLE
fix formatting and improved config examples for MM2 tuning

### PR DESCRIPTION
**Pull request by runwuf**
* updated better examples for topic black list.
* updated various formatting for readability and ability to copy code example.

### CODE REVIEW

File: docs/products/kafka/kafka-mirrormaker/concepts/mirrormaker2-tuning.rst [hunk 1]
```diff
@@ -8,22 +8,32 @@ MirrorMaker 2 common parameters
     * ``kafka_mirrormaker.emit_checkpoints_interval_seconds``
     * ``kafka_mirrormaker.sync_group_offsets_interval_seconds`` 
     * Replication flow:
-        * Sync interval in seconds
+        * ``Sync interval in seconds``
 
-3. Add `*[\-\.]` internal to the topic black list:
-    * connect.* __.*, crowdcontrol.public.events, .*[\-\.]internal
-    * The following parameters can be tuned depending on use case:
+3. Add internal topics to the topic black list:
```
Comment (staceysalamon-aiven) [inline docs/products/kafka/kafka-mirrormaker/concepts/mirrormaker2-tuning.rst:13]: Is "blacklist" the term used in the UI? If not, could we change this to "denylist" (or another rewording, as appropriate) please? For reference, see the [style guide entry for blacklist](https://developers.google.com/style/word-list#blacklist).

Comment (runwuf) [reply docs/products/kafka/kafka-mirrormaker/concepts/mirrormaker2-tuning.rst:13]: @staceysalamon-aiven 
I actually thought about this as I was updating, unfortunately it is in the UI, should we update it for now but make a change request to our console UI?
image

**Comment by runwuf** [comment]
@staceysalamon-aiven 
perhaps we should request a change to console UI?
image

**Comment by staceysalamon-aiven** [comment]
> @staceysalamon-aiven perhaps we should request a change to console UI? image


Ah, yes, if it's in the UI then we'll have to stick with it here. cc @harshini-rangaswamy since this is your area - what do you think about getting the UI text changed? Is that something you could run by that team?

**Comment by harshini-rangaswamy** [comment]
> Ah, yes, if it's in the UI then we'll have to stick with it here. cc @harshini-rangaswamy since this is your area - what do you think about getting the UI text changed? Is that something you could run by that team?

Sure, I can run this by the team. I'll keep you updated on the progress.

**Comment by runwuf** [comment]
@staceysalamon-aiven can this one be approved and merged for now? I will update it again once the console UI has been changed.

**Comment by staceysalamon-aiven** [comment]
> @staceysalamon-aiven can this one be approved and merged for now? I will update it again once the console UI has been changed.

@runwuf As this is Kafka, @harshini-rangaswamy would be best placed to do this review. Thanks!

**Review by harshini-rangaswamy — APPROVED** [review]
@runwuf Made some slight changes and formatting to the content. The rest looks good. Thanks.

**Pull request merged**